hack/deploy

```diff
@@ -37,9 +116,32 @@ pushimage() {
   ok
 }
 
+
+echo "$(deployment_target) deployment"
```

Comment:
A bit concerned about blossoming complexity of this script, but at least as a style issue, let's avoid executable code between function declarations (ref: Google's bash style guide for [Function Location](https://google.github.io/styleguide/shell.xml?showone=Function_Location#Function_Location)).